aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -307,18 +307,18 @@ The following table contains a set of common mistakes found in messaging, along
 
 Implement effective communications with these conventions.
 
-## Localization
+## Developer Documentation
```

Comment:
??

```suggestion
## Documenting Messages
```

@MikeKingdom might have good perspective on this section.

Reply:
Thanks for the feedback!